src/utils/issue.ts

```diff
@@ -224,6 +224,24 @@ export async function getAllPullRequestReviews(context: Context, pullNumber: num
   }
 }
 
+async function getReviewRequestsTimeline(context: Context, pullNumber: number, owner: string, repo: string) {
+  try {
+    const events = (await context.octokit.paginate(`GET /repos/${owner}/${repo}/issues/${pullNumber}/timeline`, {
```

Comment:
you can use `octokit.rest.issues.listEventsForTimeline` instead of hard coded string

Reply:
alright done